types/react-js-pagination/react-js-pagination-tests.tsx

```diff
@@ -8,16 +8,20 @@ class ReactPagination extends React.Component<{}, {activePage: number}> {
 			activePage: 1
 		};
 		this.handleChange = this.handleChange.bind(this);
+		this.makePageUrl = this.makePageUrl.bind(this);
 	}
 	handleChange(pageNumber: number) {
 		this.setState({
 			activePage: pageNumber
 		});
 	}
+	makePageUrl(pageNumber: number) {
+		return `#;page=${pageNumber}`
```

Comment:
```suggestion
		return `#;page=${pageNumber}`;
```

Unfortunately, build failed (https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/559358776?utm_source=github_status&utm_medium=notification) … It can help.